VERSION.txt

```diff
@@ -1 +1 @@
-v2.0.6
+v2.0.7
```

Comment:
if you set this to 2.0.8 then you won't have to change it in the next PR

Reply:
true!!!